### ISSUE TITLE
handle wildcards on Windows CMD

### ISSUE THREAD

**Issue by VladimirAlexiev** [sharkdp/bat#309]
When I try this from Windows CMD:
```bat
>cd c:\prog\bin
>bat *.pl
[bat error]: The filename, directory name, or volume label syntax is incorrect. (os error 123)
```
The same error if I give a path. So it seems to me it doesn't handle the wildcard at all.

**Comment by VladimirAlexiev** [comment]
@sharkdp I'm not a linux or cygwin programmer, but most cygwin programs support wildcard expansion, so I guess that's easy to do on cygwin. I don't think each linux program was individually ported to cygwin. Eg here's `ls` and `grep` running in CMD.
```
>cd \prog\bin

>ls *.pl
cleanup-RFC.pl  filever-ren.pl    lib2txt.pl   pfind.pl   prefixize.pl  reg-hack.pl  rncfix.pl   search.pl
exetype.pl      html-unescape.pl  perlglob.pl  pl2bat.pl  rdfpretty.pl  riotval.pl   runperl.pl  xmlpp.pl

>grep -i -c perl *.pl
cleanup-RFC.pl:1
exetype.pl:3
filever-ren.pl:1
html-unescape.pl:1
lib2txt.pl:3
perlglob.pl:9
pfind.pl:15
pl2bat.pl:63
prefixize.pl:2
rdfpretty.pl:2
reg-hack.pl:1
riotval.pl:1
rncfix.pl:1
runperl.pl:15
search.pl:25
xmlpp.pl:1
```

**Comment by sharkdp** [comment]
Why are you switching the discussion to Cygwin now?

Cygwin (I believe) uses bash as a shell which will do a proper wildcard expansion. My guess would be that `bat *.pl` would also work in the cygwin shell?

**Comment by VladimirAlexiev** [comment]
Cygwin is not related to bash. Cygwin is a set of libraries for easy porting of unix programs on windows.
All cygwin ports correctly expand wildcards in CMD, so I guess it's really easy to do this (just a matter of proper building).

**Comment by sharkdp** [comment]
So you are calling "cygwin programs" from CMD? In this case, I guess there *are* special versions which are built for cygwin because CMD definitely doesn't do any wildcard expansion.

As I said above, I think there is a really easy way to fix this by using the [wild crate](https://crates.io/crates/wild). I'm going to turn this into a feature request.

**Comment by sharkdp** [comment]
Released in [v0.8.0](https://github.com/sharkdp/bat/releases/tag/v0.8.0). @VladimirAlexiev Could you please test this?
